README.md

```diff
@@ -419,10 +424,26 @@ The following `opts` are generic in the sense that they are the same for all vis
 - `opts.margintop`   : top margin (in pixels)
 - `opts.marginbottom`: bottom margin (in pixels)
 
-
 The other options are visualization-specific, and are described in the
 documentation of the functions.
 
+### Others
+
+#### vis.close
+
+This function closes a specific window. It takes input window id `win` and environment id `eid`. Use `win` as `None` to close all windows in an environment.
+
+#### vis.win_exists
+
+This function returns a bool indicating whether or not a window `win` exists on the server already. Returns None if something went wrong. 
+
+Optional arguments:
+- `env`: Environment in which window is to be searched. Default is `None`.
```

Comment:
Small description preference nit:
Environment to search for the window in. Default is `None`.

Reply:
Addressed.